extension/src/popup/locales/en/translation.json

```diff
@@ -6,6 +6,7 @@
   "A Stellar wallet for every website": "A Stellar wallet for every website",
   "About": "About",
   "Account": "Account",
+  "Account details": "Account details",
```

Comment:
Sorry you're getting so many of these unrelated translation updates in your PR's. There's a github action that's supposed to add these automatically when devs open their PR's, but apparently it stopped working? I'll take a look.

In the future, you should only have added translations for any strings that you updated in your PR